sweep/sweeper.go

```diff
@@ -237,9 +246,23 @@ func (s *UtxoSweeper) SweepInput(input Input) (chan Result, error) {
 
 // collector is the sweeper main loop. It processes new inputs, spend
 // notifications and counts down to publication of the sweep tx.
-func (s *UtxoSweeper) collector() {
+func (s *UtxoSweeper) collector() error {
```

Comment:
If we get the best block, and also the block epoch subscription above in `Start()`, then we can ensure that if we fail to obtain these, then the sweeper doesn't start at all.

Reply:
Changed